src/components/landing/faq.astro

```diff
@@ -0,0 +1,40 @@
+---
+import { getCollection } from "astro:content";
+import { markdown } from "@/components/md";
+
+const faqs = await getCollection("faqs");
+const faqContent = await Promise.all(
```

Comment:
This will randomly organize the FAQs based on the iteration order of whatever collection astro.content.getCollection() gives us.

We need the order preserved and prescriptive in the site sources.

Reply:
I'll go throw a priority value on it sort highest first